taxonomy-editor-frontend/src/pages/editentry/ListEntryChildren.jsx

```diff
@@ -36,8 +35,8 @@ const ListEntryChildren = ({url, setUpdateNodeChildren}) => {
     }, [incomingData, setUpdateNodeChildren])
 
     // Helper functions for Dialog component
-    function handleClose() { setOpen(false); }
-    function handleOpen() { setOpen(true); }
+    function handleCloseDialog() { setOpenDialog(false); }
+    function handleOpenDialog() { setOpenDialog(true); }
```

Comment:
we are mixing `function  ... ()` with `const ... = () => {}`. Let's stick with `const .. = () => {}`, just for consistency